static/js/app.js

```diff
@@ -158,7 +180,7 @@ $('#add-label').on('click', function () {
 });
 
 //change the document's label context
-$('#label-list').on('click', '.label', function () {
+$('#label-list').on('mouseup', '.label', function () {
```

Comment:
I created a variable for this already, you should refactor a little to use it

Reply:
updated